src/components/spaces/cell-zoom/cell-mouse.ts

```diff
@@ -1,16 +1,41 @@
 import { types, Instance } from "mobx-state-tree";
```

Comment:
How come the cell-mouse model is in src/components?

Reply:
You're right it totally shouldn't be! Let me fix that in my new branch, just in case I can avoid some conflicts with this one